docs/source/plugins/plugins.rst

```diff
@@ -91,6 +91,37 @@ Available configuration options
             default_tags:
               something_default: 'The Default Tag'
 
+Prometheus Monitoring
+---------------------
+
+.. content-tabs:: left-col
+
+    The :py:class:`~tardis.plugins.prometheusmonitoring.PrometheusMonitoring` implements an interface to monitor the
+    number of drones in the states ``Booting``, ``Running``, ``Stopped``, ``Deleted``, and ``Error``.
+
+Available configuration options
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. content-tabs:: left-col
+
+    +----------------+---------------------------------------------------------------------------+-----------------+
+    | Option         | Short Description                                                         | Requirement     |
+    +================+===========================================================================+=================+
+    | addr           | Address on which the metrics are served on for Prometheus                 |  **Required**   |
+    +----------------+---------------------------------------------------------------------------+-----------------+
+    | port           | Port on which the metrics are served on for Prometheus                    |  **Required**   |
+    +----------------+---------------------------------------------------------------------------+-----------------+
+
+.. content-tabs:: right-col
+
+    .. rubric:: Example configuration
+
+    .. code-block:: yaml
+
+        Plugins:
+          PrometheusMonitoring:
+            port: 8092
```

Comment:
Since `addr` is an required option too, it should be present in the example configuration as well. Could you fix that, please?

Reply:
Good catch! Fixed in d17a7b8.